MekHQ/src/mekhq/MHQStaticDirectoryManager.java

```diff
@@ -93,6 +100,67 @@ private static void initializeAwardIcons() {
             }
         }
     }
+
+    /**
+     * Parses MekHQ's storyarcs icon folder when first called or when it was refreshed.
+     *
+     * @see #refreshStorySplash()
+     */
+    private static void initializeStorySplash() {
+        // Read in and parse MekHQ's force icon folder only when first called or when refreshed
+        if (parseStorySplashDirectory) {
+            // Set parseForceIconDirectory to false to avoid parsing repeatedly when something fails
+            parseStorySplashDirectory = false;
```

Comment:
I'm not seeing where this boolean would get reset so that a refresh call would re-load the directory items.

Reply:
NM, just lower down.